docs/user-guide/relay-nodes.md

```diff
@@ -23,3 +40,70 @@ If you're using the Nexodus rpm package, edit `/etc/sysconfig/nexodus` to add th
 ```sh
 NEXD_ARGS="--service-url https://try.nexodus.io relay"
 ```
+
+## Setup Self Hosted Nexodus DERP Relay
+
+If user would prefer to use it's own relay instead of the public DERP relay, user can deploy the relay node on their own infrastructure and on-board it to Nexodus. Peers behind symmetric NAT will switch to the self hosted relay once it's successfully on-boarded. DERP relay uses TLS for communication, so user will need to provide a TLS certificate and key to the relay node. User can onboard the relay in following two ways:
```

Comment:
```suggestion
If the user would prefer to use its own relay instead of the public DERP relay, the user can deploy the relay node on their own infrastructure and on-board it to Nexodus. Peers behind symmetric NAT will switch to the self-hosted relay once it's successfully on-boarded. DERP relay uses TLS for communication, so the user will need to provide a TLS certificate and key to the relay node. Users can onboard the relay in the following two ways:
```